public/app/types/alerting.ts

```diff
@@ -142,6 +142,36 @@ export interface NotificationChannelState {
   notificationChannel: any;
 }
 
+export interface IntegrationError {
+  lastError: string;
+  lastNotify: DateTime;
```

Comment:
Does it come from the `API`? If yes, then this should probably be of type `string`

Reply:
do you mean lastNotify?, yes

Reply:
Yes, I mean `lastNotify`. It is a JSON object deserialized from the server response, so the type will be rather `string` than `DateTime`